library-best-practices/JLBP-4.md

```diff
@@ -0,0 +1,44 @@
+[JLBP-4] Avoid dependencies on unstable libraries and features
+--------------------------------------------------------------
+
+- Unstable libraries are defined as libraries that allow for breaking changes to
```

Comment:
"Unstable libraries are defined as libraries that allow for" --> Unstable libraries allow